### ISSUE TITLE
WCAG 2.2 SC: part of the SC text is missing

### ISSUE THREAD

**Issue by remibetin** [w3c/wai-wcag-em-report-tool#196]
[1.4.13](https://www.w3.org/TR/WCAG22/#content-on-hover-or-focus):
> Exception: The visual presentation of the additional content is controlled by the [user agent](https://www.w3.org/TR/WCAG22/#dfn-user-agents) and is not modified by the author.

[2.4.13](https://www.w3.org/TR/WCAG22/#focus-appearance):

> - is at least as large as the area of a 2 [CSS pixel](https://www.w3.org/TR/WCAG22/#dfn-css-pixels) thick [perimeter](https://www.w3.org/TR/WCAG22/#dfn-perimeter) of the unfocused component or sub-component, and
> - has a contrast ratio of at least 3:1 between the same pixels in the focused and unfocused states.
> 
> Exceptions:
> 
> - The focus indicator is determined by the [user agent](https://www.w3.org/TR/WCAG22/#dfn-user-agents) and cannot be adjusted by the author, or
> - The focus indicator and the indicator's background color are not modified by the author.

[2.5.8](https://www.w3.org/TR/WCAG22/#target-size-minimum):

> Spacing
> Undersized targets (those less than 24 by 24 CSS pixels) are positioned so that if a 24 CSS pixel diameter circle is centered on the [bounding box](https://www.w3.org/TR/WCAG22/#dfn-bounding-boxes) of each, the circles do not intersect another target or the circle for another undersized target;
> Equivalent
> The function can be achieved through a different control on the same page that meets this criterion;
> Inline
> The target is in a sentence or its size is otherwise constrained by the line-height of non-target text;
> User Agent Control
> The size of the target is determined by the [user agent](https://www.w3.org/TR/WCAG22/#dfn-user-agents) and is not modified by the author;
> Essential
> A particular [presentation](https://www.w3.org/TR/WCAG22/#dfn-presentation) of the target is [essential](https://www.w3.org/TR/WCAG22/#dfn-essential) or is legally required for the information being conveyed.

[3.2.6](https://www.w3.org/TR/WCAG22/#consistent-help):
> - Human contact details;
> - Human contact mechanism;
> - Self-help option;
> - A fully automated contact mechanism.

[3.3.7](https://www.w3.org/TR/WCAG22/#redundant-entry):
> - auto-populated, or
> - available for the user to select.
> 
> Except when:
> 
> - re-entering the information is [essential](https://www.w3.org/TR/WCAG22/#dfn-essential),
> - the information is required to ensure the security of the content, or
> - previously entered information is no longer valid.

[3.3.8](https://www.w3.org/TR/WCAG22/#accessible-authentication-minimum):
> Alternative
> Another authentication method that does not rely on a cognitive function test.
> Mechanism
> A [mechanism](https://www.w3.org/TR/WCAG22/#dfn-mechanism) is available to assist the user in completing the cognitive function test.
> Object Recognition
> The cognitive function test is to recognize objects.
> Personal Content
> The cognitive function test is to identify [non-text content](https://www.w3.org/TR/WCAG22/#dfn-non-text-content) the user provided to the website.

[3.3.9](https://www.w3.org/TR/WCAG22/#accessible-authentication-enhanced):

> Alternative
> Another authentication method that does not rely on a cognitive function test.
> Mechanism
> A [mechanism](https://www.w3.org/TR/WCAG22/#dfn-mechanism) is available to assist the user in completing the cognitive function test.
